crypto/blake2/build.info

```diff
@@ -0,0 +1,3 @@
+LIBS=../../libcrypto
+SOURCE[../../libcrypto]=\
+        blake2b.c blake2s.c {- $target{blake2b_obj} -}
```

Comment:
I realize I will need to do a bit of work (modify Configure, build.info,
Makefile.in, maybe a couple more), but I prefer to keep this initial PR
simple.  Thanks for reviewing it carefully!  I do make mistakes, and I feel
a lot better now that a few people have taken a good look.

On Sun, Feb 28, 2016 at 11:30 PM, Richard Levitte <notifications@github.com>
wrote:

> In crypto/blake2/build.info
> <https://github.com/openssl/openssl/pull/566#discussion_r54375731>:
>
> > @@ -0,0 +1,3 @@
> > +LIBS=../../libcrypto
> > +SOURCE[../../libcrypto]=\
> > +        blake2b.c blake2s.c {- $target{blake2b_obj} -}
>
> $target{blak2_obj} is fine for the older Makefile.in. The new build.info
> files deal with source, and therefore use the names from
> Configurations/00-base-templates.conf directly. That's where the asm
> variable comes from.
>
> I noticed you removed it entirely, which is technically fine now, but will
> not be fine as soon as asm source is added.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openssl/openssl/pull/566/files#r54375731>.
>